dev-docs/bidders/yahoossp.md

```diff
@@ -7,19 +7,18 @@ pbjs: true
 media_types: banner, video
 biddercode: yahoossp
 prebid_member: true
-gdpr_supported: false
+gdpr_supported: true
 usp_supported: true
+gpp_supported: true
 schain_supported: true
 coppa_supported: true
+floors_supported: true
+fpd_supported: true
```

Comment:
I can't let you flip the FPD flag to true unless you specifically detail in the doc precisely what you support. e.g. site.ext.data? site.content.data? user.ext.data? etc.